user_docs/en/userGuide.t2t

```diff
@@ -2231,6 +2231,16 @@ Please see the [EcoBraille documentation ftp://ftp.once.es/pub/utt/bibliotecnia/
 | Toggle braille tethered to | A |
 %kc:endInclude
 
+++ Super Braille ++

```

Comment:
This heading calls it "Super Braille" (with a space), whereas elsewhere, it's "SuperBraille" (camel case). Which one? :)